### PR TITLE
remove indirect model imports from world_vision/__init__.py

### PR DESCRIPTION
**Pull request by nickpell**

### CODE REVIEW

**Comment by nickpell** [comment]
@sravfeyn @dannyroberts

File: corehq/apps/reports/views.py [hunk 1]
```diff
@@ -200,9 +201,9 @@ def can_view_attachments(request):
 
 @login_and_domain_required
 def default(request, domain):
-    module = Domain.get_module_by_name(domain)
-    if hasattr(module, 'DEFAULT_REPORT_CLASS'):
-        return HttpResponseRedirect(getattr(module, 'DEFAULT_REPORT_CLASS').get_url(domain))
+    if domain == 'wvindia2' and get_domain_module_map().get('wvindia2'):
```
Comment (sravfeyn) [inline corehq/apps/reports/views.py]: Why not define `wvindia2` as a constant and reuse it?

Comment (nickpell) [inline corehq/apps/reports/views.py]: good point, will do

Comment (nickpell) [inline corehq/apps/reports/views.py]: updated

**Comment by nickpell** [comment]
✅

**Pull request merged**